ilearn/onderwijsstructuur-extension/schemes/onderwijsstructuur.ttl

```diff
@@ -0,0 +1,634 @@
+@prefix ocol: <https://w3id.org/onderwijs-vlaanderen/id/collectie/> .
```

Comment:
Is this file still relevant? can't it be removed?

Reply:
Done, see https://github.com/i-Learn-SKOS/common-conceptschemes/pull/7/commits/ccd84267e436482ff3050202e9df7e8056656e38